files/verbs/all.txt

```diff
@@ -151,6 +151,7 @@ glut                     The glut utility library for OpenGL (Mark J. Kilgard, 2
 gmdls                    General MIDI DLS Collection (Microsoft / Roland, 1999) [downloadable]
 hid                      MS hid (Microsoft, 2003) [downloadable]
 icodecs                  Indeo codecs (Intel, 1998) [downloadable]
+cinepak                  Cinepak Codec (Radius, 1995) [downloadable]
```

Comment:
That should be sorted (though you can leave it out, it's updated in release process).

Reply:
done